src/lib/routes/admin-api/index.ts

```diff
@@ -101,6 +102,13 @@ class AdminApi extends Controller {
             '/constraints',
             new ConstraintsController(config, services).router,
         );
+        this.app.use(
```

Comment:
is publicSignup the best wording for this feature? What about inviteLink?

Reply:
Better, thanks!